src/containers/CreateInstructor/index.jsx

```diff
@@ -1,21 +1,34 @@
 import { connect } from 'react-redux';
 
 import CreateInstructorPage from '../../components/CreateInstructorPage';
-import { createInstructor } from '../../data/actions/instructorInfo';
+import { fetchInstructorInfo, createInstructor } from '../../data/actions/instructorInfo';
 import { fetchInstructorOptions } from '../../data/actions/instructorOptions';
 
 const mapStateToProps = state => ({
-  publisherUserInfo: state.publisherUserInfo,
+  publisherUserInfo: state.publisherUserInfo,  // BJH: can we ditch this?
   instructorOptions: state.instructorOptions,
   instructorInfo: state.instructorInfo,
 });
 
+const mergeProps = (stateProps, actionProps, { id }) => {
+  console.log('======  Id: ');
+  console.log(id);  // BJH: DELETE
+
+  return {
+    ...stateProps,
+    fetchInstructorInfo: () => actionProps.fetchInstructorInfo(id),
```

Comment:
I don't think you need to have this function in `mergeProps`

Reply:
Removed